src/design-system/InputText.tsx

```diff
@@ -17,15 +17,17 @@ const TextInputContainer = styled.div`
 `;
 
 const InputWrapper = styled.div`
-  ${StyledInput}
```

Comment:
This is where my previous misunderstanding was that caused the bug.

I thought that including this would bring in the styles from `StyledInput`, but all it did was bring in the auto-generated class name!

The correct way to do this is to use `styled.css` to create a stylesheet (in the fixed version, named `InputStyle`). This stylesheet can then be brought in this way.